pkg/microservice/aslan/core/service/service/helm.go

```diff
@@ -430,33 +477,48 @@ func CreateOrUpdateHelmServiceFromGitRepo(projectName string, args *HelmServiceC
 
 			log.Info("Found valid chart, Starting to save and upload files")
 
+			rev, err := getNextServiceRevision(projectName, serviceName)
+			if err != nil {
+				log.Errorf("Failed to get next revision for service %s, err: %s", serviceName, err)
+				finalErr = e.ErrCreateTemplate.AddErr(err)
+				return
```

Comment:
`return finalErr`?

Reply:
this is no return value for this function, finalErr is handled in defer function